cmd/vulcan-exposed-ftp/main.go

```diff
@@ -109,30 +99,53 @@ func exposedFTP(target string, nmapReport *gonmap.NmapRun) []report.Vulnerabilit
 				"Service":  port.Service.Product,
 				"Version":  port.Service.Version,
 			}
+			gr := report.ResourcesGroup{
+				Name: "Network Resources",
+				Header: []string{
+					"Hostname",
+					"Port",
+					"Protocol",
+					"Service",
+					"Version",
+				},
+			}
 			gr.Rows = append(gr.Rows, networkResource)
+			v.Resources = []report.ResourcesGroup{gr}
+			v.ID = computeVulnerabilityID(target, v.AffectedResource, port.Service.Product, port.Service.Version, v.Score, "exposed")
```

Comment:
Now there's a new `Vulnerability.Fingerprint` attributed to be used instead of the `ID`. Also we are not adding the target nor the affected resource to calculate the fingerprint now.

Suggested change:
```
v.Fingerprint = helpers.ComputeFingerprint(port.Service.Product, port.Service.Version)
```
(Adding the score and the static word it's harmless and we can keep it if you want)

Reply:
If you agree I'll keep the static word to differentiate between the different issues this check can find.